Makefile

```
@@ -0,0 +1,61 @@
+.PHONY: help dev build clean logs prod-deploy
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)